pymatgen/electronic_structure/dos.py

```diff
@@ -1165,6 +1166,150 @@ def get_upper_band_edge(
         upper_band_edge = energies[np.argmax(densities)]
         return upper_band_edge
 
+    def get_dos_fp(self, type="summed_pdos", binning=True, min_e=None, max_e=None, nbins=256, normalize=True):
+        """
+        Generates the DOS fingerprint based on work of
+        F. Knoop, T. A. r Purcell, M. Scheffler, C. Carbogno, J. Open Source Softw. 2020, 5, 2671.
+        Source - https://gitlab.com/vibes-developers/vibes/-/tree/master/vibes/materials_fp
+        Copyright (c) 2020 Florian Knoop, Thomas A.R.Purcell, Matthias Scheffler, Christian Carbogno
+
+
+        Args:
+            type (str): Specify fingerprint type needed can accept 's/p/d/f/summed_pdos/tdos'
+            min_e (float): The minimum mode energy to include in the fingerprint
+            max_e (float): The maximum mode energy to include in the fingerprint
+            nbins (int): Number of bins to be used in the fingerprint
```

Comment:
Can we rename this to `n_bins` for proper snake casing?

Reply:
Done 😃